src/styles/_variables.scss

```diff
@@ -1,4 +1,5 @@
 @import '_sky-variables.scss';
+@import '_font-awesome-variables';
```

Comment:
It would be better to pull the variables directly from the node_modules folder. (I added the SKY UX variables to our source because there's no way to get the variables dynamically, as of yet.)